### PR TITLE
chore(tokens): build tori tokens with finn color palette

### PR DESCRIPTION
**Pull request by BalbinaK**
Just like with `/tokens/blocket.se`, for now we populate Tori tokens with Finn colors. In our source of truth (the Taxonomy spreadsheet) the [Tori color palette](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=1864511726) is almost complete ([related design task](https://trello.com/c/Ns95RtKZ/101-create-a-core-tori-color-palette-50-900s)) but the respective [Tori color values](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=1588196525&range=E:E) are not yet assigned to any semantic tokens. It'll be handled in [this design task](https://trello.com/c/YBCKwC2j). Once both are ready, we will update the .yml files in `/tokens/tori.fi` directory ([related dev task](https://trello.com/c/8AFjxhO5)).

### CODE REVIEW

File: tokens/tori.fi/colors.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+token: defs
+
+white:
+  100: "#ffffff"
+black:
+  100: "#000000"
+transparent: transparent
```
Comment (magnuh) [inline tokens/tori.fi/colors.yml:8]: Is this really needed? Shouldn't transparent always be transparent? Or do we want the possibility to have transparent as some kind of semi-transparent in any brands? 😄

File: tokens/tori.fi/colors.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+token: defs
+
+white:
+  100: "#ffffff"
+black:
+  100: "#000000"
```
[the remark this answers is not in the document]
Comment (BalbinaK) [reply tokens/tori.fi/colors.yml:7]: @adidick

[the remark this answers is not in the document]
Comment (BalbinaK) [reply tokens/tori.fi/colors.yml]: From a [separate discussion](https://sch-chat.slack.com/archives/C050N3NQ204/p1680078944297589) it seems it was agreed that black and white can be assigned "black" and "white" values, so we don't do `black-100: #000000` but `black: black`. Fixed in [4c65a2a](https://github.com/warp-ds/tokens/pull/18/commits/4c65a2abf0585195b2ee89ac87ab6a3872cdbb82)

File: tokens/tori.fi/foundation.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+token: maps
+
+text:
+  color:
+    _: red-700
+    subtle: gray-500
+    placeholder: bluegray-300
+    inverted:
+      _: white-100
```
Comment (AnnaRybkina) [inline tokens/tori.fi/foundation.yml:10]: Should this be `white`?
```suggestion
      _: white
```

**Review by AnnaRybkina — APPROVED** [review]

File: tokens/tori.fi/foundation.yml (hunk 3 above)
Comment (BalbinaK) [reply tokens/tori.fi/foundation.yml]: Thanks! Fixed all cases in [1ebe4a4](https://github.com/warp-ds/tokens/pull/18/commits/1ebe4a4ad53d769b506a349b6827fce12ee01970)

**Pull request merged**